### PR TITLE
Add some comments and updated the random generator (Correct)

### PR DESCRIPTION
**Pull request by qwqAutomaton**

### CODE REVIEW

File: src/pfUI.cpp [hunk 1]
```diff
@@ -114,19 +114,19 @@ void BlinkCoord(short ax, short ay, bool signDir) {
 	setColor(lightGrey, dbc);
 	gotoXY((scrW - tmp2.length()) / 2, 9);
 	cout << tmp2;
-	Sleep(rand() % 250);
+	Sleep(rng() % 250);
```
Comment (Zjl37) [inline src/pfUI.cpp:118]: `rng` is not declared here and will cause compile error.
Please at lease compile before you commit!

**Review by Zjl37 — CHANGES_REQUESTED** [review]

File: src/pfUI.cpp (hunk 1 above)
Comment (qwqAutomaton) [reply src/pfUI.cpp:118]: fixed qwq

**Comment by qwqAutomaton** [comment]
I suggest that we should change the version to "2.4.1" because there is small changes (on `rng`).

**Comment by qwqAutomaton** [comment]
major version 2, sub version 4, small fix 1 (or sth else

**Comment by Zjl37** [comment]
> 
> 
> I suggest that we should change the version to "2.4.1" because there is small changes (on `rng`).

This will be part of v2.6, because I've already used tag v2.4 and v2.5 somewhere else. 😃

**Review by Zjl37 — APPROVED** [review]

**Pull request merged**